node/index.ts

```diff
@@ -11,6 +11,58 @@ const SC = bindings('libsignal_client_' + os.platform()) as typeof SignalClient;
 
 export const { initLogger, LogLevel } = SC;
 
+export class Aes256GcmSiv {
+  private readonly nativeHandle: SignalClient.Aes256GcmSiv;
+
+  constructor(key: Buffer) {
+    this.nativeHandle = SC.Aes256GcmSiv_New(key);
+  }
+
+  encrypt(message: Buffer, nonce: Buffer, associated_data: Buffer): Buffer {
+    return SC.Aes256GcmSiv_Encrypt(
+      this.nativeHandle,
+      message,
+      nonce,
+      associated_data
+    );
+  }
+
+  decrypt(message: Buffer, nonce: Buffer, associated_data: Buffer): Buffer {
+    return SC.Aes256GcmSiv_Decrypt(
+      this.nativeHandle,
+      message,
+      nonce,
+      associated_data
+    );
+  }
+}
+
+export class ProtocolAddress {
+  private readonly nativeHandle: SignalClient.ProtocolAddress;
+
+  private constructor(handle: SignalClient.ProtocolAddress) {
+    this.nativeHandle = handle;
+  }
+
+  static fromNativeHandle(
+    handle: SignalClient.ProtocolAddress
+  ): ProtocolAddress {
+    return new ProtocolAddress(handle);
+  }
+
+  static new(name: string, deviceId: number): ProtocolAddress {
+    return new ProtocolAddress(SC.ProtocolAddress_New(name, deviceId));
+  }
+
+  name(): string {
```

Comment:
I think we already checked with the Desktop folks over whether to use method or property syntax for things like this, but it's worth double-checking.

Reply:
Evan's comment (in #124) on this "I'd call this getPublicKey and keep it as a function. I rarely see accessors used in JavaScript-land. (This is anecdotal, and other Desktop devs might have different experiences.)"